patch/tree-sitter-0.22.6/Makefile

```diff
@@ -0,0 +1,113 @@
+VERSION := 0.22.6
```

Comment:
would be better to actually store a patch in a patch/  directory, instead of the whole Makefile, which is then easier to see what we actually change.

Reply:
diff -u a/Makefile b/Makefile > tree_sitter.patch
and then 
cd tree-sitter_path; patch -p1 < /path/to/tree_sitter.patch

Reply:
ok. It was combination of laziness and lack for familiarity with patches.